seeds/swapData.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "title": "Strawberry Seedlings",
+    "description": "Many stawberry seedlings to swap, will swap for other berries or fruit",
+    "status": "Swap"
+  },
```

Comment:
This is missing `date_created` and `user_id` fields

Reply:
Yes mate, those will be auto incremented.

Reply:
@jnaoroji No the `id` of the Swap is auto-incremented, not the `date_created` and `user_id` which are both mandatory fields per your model definition.